spec/features/sign_out_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.feature 'Sign Out', type: :feature, js: true do
-  let(:user) { FactoryGirl.create(:user) }
+  let(:user) { FactoryGirl.create( :user, first_login: true, first_target: true ) }
```

Comment:
Your user factory already have those values by default right? You don't need to override them in this statement.